skorch/utils.py

```diff
@@ -768,3 +769,16 @@ def _check_f_arguments(caller_name, **kwargs):
             key = 'module_' if key == 'f_params' else key[2:] + '_'
             kwargs_module[key] = val
     return kwargs_module, kwargs_other
+
+
+def check_torch_weights_only_default_true():
```

Comment:
Given how specific this function is to `torch.load`, can this return `torch_load_kwargs` itself?

Reply:
Good point, I made the suggested change.